src/challenges.js

```diff
@@ -38,6 +58,18 @@ const checkMatchingLeaves = obj => {
 };
 
 const flatten = elements => {
+  const newArr = [];
```

Comment:
flatten function in this examples is equivalent to flat() method that creates new array with sub-array elements concatenated.